### ISSUE TITLE
Hierarchy

### ISSUE THREAD

**Issue by bhack** [lgeiger/ide-python#61]
I still don't see class methods.

https://github.com/palantir/python-language-server/pull/176

**Comment by lgeiger** [comment]
What version of the `python-language-server` have you installed?

**Comment by bhack** [comment]
I've tested with 0.12.1

**Comment by lgeiger** [comment]
Are you using the latest versions of `ide-python` and `atom-ide-ui` too?

**Comment by bhack** [comment]
Yes

**Comment by lgeiger** [comment]
Strange, it's working on my machine. Can you provide example code?

**Comment by bhack** [comment]
Can you give me a sample that it is working on your side? Cause I've already tested with multiple files.

**Comment by lgeiger** [comment]
Checkout:
```python
class Bag:
    def __init__(self):
        self.data = []

    def add(self, x):
        self.data.append(x)

    def addtwice(self, x):
        self.add(x)
        self.add(x)
```

It should look similar to this:
<img width="819" alt="bildschirmfoto 2017-12-29 um 02 14 37" src="https://user-images.githubusercontent.com/13285808/34427074-1be7d3b0-ec3e-11e7-8186-def9c4c11ceb.png">

**Comment by lenlen** [comment]
I have the same problem, I can't see the class methods with python-language-server 0.12.1. Here the screenshot
![image](https://user-images.githubusercontent.com/2477675/34436721-b0886642-ec98-11e7-96c3-484bb6685b7e.png)

**Comment by lgeiger** [comment]
Which version of atom are you using?

**Comment by lenlen** [comment]
1.23.1

**Comment by lgeiger** [comment]
Ahh got it. You have to enable the `Jedi Symbols: All Scopes`  setting in the `ide-python` package.

Sorry for not figuring this out sooner.

Do you think we should enable this behavior by default? I'd be happy to receive a PR changing that.

**Comment by lenlen** [comment]
Great, now it works, thanks! Maybe it would be useful to have this setting enabled by default.

**Comment by lgeiger** [comment]
> Great, now it works, thanks! Maybe it would be useful to have this setting enabled by default.

👍Want to send a PR to enable this?

**Comment by lenlen** [comment]
Here the pull request https://github.com/lgeiger/ide-python/pull/62

**Comment by lgeiger** [comment]
Closed by #62
